src/style.css

```diff
@@ -1 +1,102 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
```

Comment:
Default value, so you can just remove it
```suggestion
```

Reply:
Done